### PR TITLE
Ext sidecars

### PR DESCRIPTION
**Pull request by akucharska**
**Description**

Changes proposed in this pull request:

- [x] Sidecars List view
- [x] Sidecars Form
   - [x] Outbound Traffic Policy Mode
   - [x] Egress
   - [x] Ingress
   - [x] Workload Selector
- [x] Sidecars Details view
   - [x] Outbound Traffic Policy Mode
   - [x] Egress
   - [x] Ingress
   - [x] Workload Selector
   - [x] Graph

**Related issue(s)**

### CODE REVIEW

File: examples/resources/configuration/istio/sidecars.yaml [hunk 1]
```diff
@@ -0,0 +1,292 @@
...
+        {
+          "widget": "Table",
+          "source": "spec.egress",
+          "name": "Egress",
+          "visibility": "$exists(*)",
+          "children": [
+            {
+              "source": "$item.port", 
+              "name": "item.port", 
+              "widget": "Panel",
```
[the remark this answers is not in the document]
Comment (qbalukom) [reply examples/resources/configuration/istio/sidecars.yaml:37]: (not blocking, just an idea)

File: examples/resources/configuration/istio/sidecars.yaml [hunk 2]
```diff
@@ -0,0 +1,292 @@
...
+              "TLS"
+            ],
+            "required": true,
+            "placeholder": "placeholders.dropdown",
+          },
+          { "path": "targetPort", "simple": true },
+        ]
+      },
+      {
+        "widget": "FormGroup",
```
[the remark this answers is not in the document]
Comment (qbalukom) [reply examples/resources/configuration/istio/sidecars.yaml:153]: (tbh, i'm surprised it even works this way...)

File: examples/resources/configuration/istio/sidecars.yaml [hunk 3]
```diff
@@ -0,0 +1,292 @@
...
+  general: |-
+    {
+      "resource": {
+        "kind": "Sidecar",
+        "group": "networking.istio.io",
+        "version": "v1alpha3"
+      },
+      "urlPath": "ext-sidecars",
+      "category": "Istio",
+      "name": "Ext Sidecars",
```
Comment (qbalukom) [inline examples/resources/configuration/istio/sidecars.yaml:258]: and just `Sidecars` for name?

File: examples/resources/configuration/istio/sidecars.yaml [hunk 4]
```diff
@@ -0,0 +1,292 @@
...
+      "scope": "namespace",
+      "description": "{{[Sidecar](https://istio.io/latest/docs/reference/config/networking/sidecar/)}} manages the incoming and outgoing communication in a workload it is attached to."
+
+    }
+  list: |-
+    [
+      { "source": "spec.outboundTrafficPolicy.mode","name": "spec.outboundTrafficPolicy.mode"},
+      { "source": "spec.workloadSelector.labels", "name": "spec.workloadSelector.labels", "widget": "Labels"}
+    ]
+  translations: |-
```
[the remark this answers is not in the document]
Comment (qbalukom) [reply examples/resources/configuration/istio/sidecars.yaml:268]: and especially when in some cases ("Egress", "Ingress", "Name" to name a few, as well as resource name, category and description) we actually use plain text in details - it should be either all use plain text, or all use keys

File: examples/resources/configuration/istio/sidecars.yaml (hunk 1 above)
[the remark this answers is not in the document]
Comment (akucharska) [reply examples/resources/configuration/istio/sidecars.yaml:37]: I think on Thursday daily we decided not to create new widget and wait for UI expert to suggest changes

File: examples/resources/configuration/istio/sidecars.yaml [hunk 5]
```diff
@@ -0,0 +1,292 @@
...
+      }
+    ]
+  general: |-
+    {
+      "resource": {
+        "kind": "Sidecar",
+        "group": "networking.istio.io",
+        "version": "v1alpha3"
+      },
+      "urlPath": "ext-sidecars",
```
[the remark this answers is not in the document]
Comment (akucharska) [reply examples/resources/configuration/istio/sidecars.yaml:256]: Gateways were also added as separate page (with ext prefix)

File: examples/resources/configuration/istio/sidecars.yaml (hunk 3 above)
Comment (akucharska) [reply examples/resources/configuration/istio/sidecars.yaml:258]: Gateways were also added as separate page (with ext prefix)

**Review by qbalukom — APPROVED** [review]

**Pull request merged**